content/docs/tutorials/cloud-tutorial.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Using InfinyOn Cloud Backend
+menu: InfinyOn Cloud Tutorial
+weight: 60
+---
+
+This tutorial will walk you through setting up an InfinyOn account, and using
+Fluvio, both as a standalone command, and as part of a larger program.
+
+## Basic Setup
+
+There are 3 main steps for setting up for the Cloud: Installing the CLI,
+registering for a Cloud account, and finally linking the two together.
+
+The following sections will walk you through how to do that.
+
+### Install Fluvio CLI
+
+Download the Fluvio CLI with the following command.
+
+%copy first-line%
+```bash
+$ curl -fsS https://packages.fluvio.io/v1/install.sh | bash
+```
+
+{{< partial file="docs/path-env.html" >}}
+
+
+### Create InfinyOn Cloud Account
+
+Head over to the [InfinyOn Cloud sign up page](https://infinyon.cloud).
+
+<img src="../images/cloud-signup.jpg"
+     alt="A screenshot of the InfinyOn new account form, with Name, Organization, Email, and Password fields"
+     style="justify: center; max-width: 300px" />
+
+After filling out the form, you'll be greeted with a success message telling you
+to verify your email. You'll need to complete this step in order to continue.
+
+~> The link in the email will time out after a few minutes.
+
+<img src="../images/cloud-verification.jpg"
+     alt="A screenshot of the verification email received after completing the signup form, including a verification link"
+     style="justify: center; max-width: 500px" />
+
+You should get a confirmation that your account is ready to use.
+
+<img src="../images/cloud-confirmation.jpg"
+     alt="A screenshot of the prompt received after clicking the verification link, saying the account is ready to use"
+     style="justify: center; max-width: 300px" />
+
+
+At this point, you can log in via the Fluvio CLI and start sending and receiving
+messages to your Fluvio cluster.
+
+### Link InfinyOn Cloud to Fluvio CLI
+
+Use the command `fluvio cloud login` to connect the InfinyOn Cloud to your
+Fluvio CLI. It will ask for your account credentials, as seen below.
+
+%copy first-line%
+```bash
+$ fluvio cloud login
+InfinyOn Cloud email: John@example.com
+Password:
+```
+
+Use the `fluvio profile list` command to confirm that your CLI is linked to the
+InfinyOn Cloud instance.
+
+%copy first-line%
+```bash
+$ fluvio profile list
+     PROFILE   CLUSTER   ADDRESS                     TLS
+  *  cloud     cloud     router.infinyon.cloud:9003  Verified
+```
+
+## A Quick Introduction to the Fluvio Interface
+
+You can now see Fluvio in action by creating a simple topic, and pushing data to it:
```

Comment:
This area seems to be a duplicate of the area below. I guess, we should describe as 3 core components: 1) Create topic, 2) produce, 3) consume. As it is now, `greetings` leaves me hanging without a consumer. .